### ISSUE TITLE
Events aren't triggered

### ISSUE THREAD

**Issue by Naamloos** [Luigifan/DiscordSharp#34]
This might have to do with changes in events at discordsharp-v0.6.0.68-dev.

**Comment by suicvne** [comment]
Specifics?

**Comment by Naamloos** [comment]
I'm sorry, 
client.PrivateMessageReceived and client.MessageReceived.

**Comment by suicvne** [comment]
Is it in the example application?

**Comment by Naamloos** [comment]
It is in my example application

**Comment by suicvne** [comment]
Okay, I'll take a look today and get back to you. 
A recent addition is a 3 second artifican delay being ran before triggering connected. looks something like this

``` csharp
Task.Run(()=>
{
    Task.Delay(3500);
    Connected?.Invoke(this, Me);
});
```

**Comment by Naamloos** [comment]
Thanks :smile:

**Comment by suicvne** [comment]
![image](https://cloud.githubusercontent.com/assets/861492/14405151/2060d888-fe56-11e5-865c-a60d647dfdd3.png)
Events are triggering fine for me. I just added a WriteLine everytime the MessageReceived event fires. No other changes to code. (I know, it's not the actual message text but i just wanted to verify this quickly)

**Comment by Naamloos** [comment]
That's exactly what i did, weird. let me try again ;)

**Comment by Naamloos** [comment]
well it works now... false alarm c: thanks for your support

i tried earlier today and the bot would respond or log any messages. I'm sorry for bothering you :/

**Comment by suicvne** [comment]
No problem! :+1: All is good :)
